### ISSUE TITLE
Update alt text for Sustainable Development Logo image on the About Us page to adhere to WCAG

### ISSUE THREAD

**Issue by Zak234** [hackforla/website#3403]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the Sustainable development logo image on the [About Us page](https://www.hackforla.org/about/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] In the file `_includes/about-page/about-card-sustainability.html`, change the following alt text

From
```
alt="Los Angeles Sustainable Development wheel logo"
```

To
```
alt=""
```

- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/about-page/about-card-accomplishments.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/a83dd650b664618ff2352b2fb93e8ae4acfb402b/_includes/about-page/about-card-sustainability.html#L56

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by kathrynsilvaconway** [comment]
@KBWells77 Hi, this issue has not been updated in over 2 weeks. If you are still interested in working with HfLA, please let us know by updating your ETA and Availability. If we don't hear back within 7 days, we will unassign you from this issue and put it back in the queue. Thanks!

**Comment by roslynwythe** [comment]
i. Availability: 10/24 - 10/26
ii. ETA: By 10/26

**Comment by roslynwythe** [comment]
Hi I've run into a problem.  Before creating my issue branch I used ‘Sync Fork’ on Github to sync my fork (roslynwythe/website) from hackforla/website.  Then several hours later when I was ready to push my local commits to the repository, I attempted to do another sync using my local CLI:
```
Roslyn Wythe@DESKTOP-CGL018T MINGW64 ~/hfla/website (update-alt-text-3403)$ git pull upstream
remote: Enumerating objects: 18, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 18 (delta 10), reused 13 (delta 8), pack-reused 0
Unpacking objects: 100% (18/18), 4.40 KiB | 32.00 KiB/s, done.
From https://github.com/hackforla/website
   b2668e25..3dc36169  gh-pages   -> upstream/gh-pages
You asked to pull from the remote 'upstream', but did not specify
a branch. Because this is not the default configured remote
for your current branch, you must specify a branch on the command line.
```

I went to GitHub to my fork page and clicked Sync Fork, and the message said there are no updates to fetch.  But in the message above it appeared that there was an update in gh-pages.  Please advise, is it safe to merge my issue branch?
